101-webapp-basic-windows/azuredeploy.json

```diff
@@ -28,6 +28,7 @@
             "type": "string",
             "defaultValue": ".net",
             "allowedValues": [
+                "none",
```

Comment:
"None" is not needed. App Service creates a .Net app when no language is specified. 
Also, if we allow a "none" value then we'll need a configReference["none"] variable. 
I can add it if you think "None" should be allowed.

Reply:
If it defaults to .NET, then it seems like allowing "none" is misleading. If someone set it to none and then later added an app, is it difficult to change the site config settings to use another language?

Reply:
@tfitzmac In regards to the siteConfig - yes it can be difficult, ex: Node.js requries an app setting rather than a portal drop-down selection. It's not intuitive and the customer wouldn't know w/o searching for it. Having this within the template will ensure those customers can create apps w/ the correct configurations.